README.md

```diff
@@ -222,11 +233,11 @@ the fact that we hit the same Endpoint definition three times. Let's refactor by
 finding the kind of `Impl Endpoint_1 IO`
 
 ```haskell
-type Impl_Endpoint =
-       IO (Response '[] 'Empty)
-  :<|> IO (Response '['H.Location '::: RawText] 'Empty)
-  :<|> IO (Response '[] ('Body '[Ct.TextPlain] Text))
-  :<|> IO NotHere
+type Impl_Endpoint_ = 
```

Comment:
That seems plausible. We can restore that bit if we need it, but things have changed there just a bit.